hera_qm/uvflag.py

```diff
@@ -473,7 +511,7 @@ def to_waterfall(self, method='quadmean', keep_pol=True):
             self.weights_array = w
             self.time_array, ri = np.unique(self.time_array, return_index=True)
             self.lst_array = self.lst_array[ri]
-        if (method == 'or') and (self.mode == 'flag'):
+        if ((method == 'or') or (method == 'and')) and (self.mode == 'flag'):
```

Comment:
Here and the next two bits were bugs

Reply:
I'm having trouble interpreting these cases. Is the idea that if you use the other averaging methods when you're in flag mode, you'll go into metric mode? Is that the most sensible behavior? Perhaps explain that in comments.